.github/workflows/test-runner.yml

```diff
@@ -0,0 +1,15 @@
+name: tests-run-on-push
```

Comment:
Il nome del file non mi convince, (forse anche il nome del workflow, ma puo' andare bene per ora), perche' per come lavoriamo noi almeno, abbiamo bisogno piu' workflows, dove tutti runnano dei test quindi tendiamo a chiamarli in modo piu' legato al loro scopo